doc/index.qdoc

```diff
@@ -17,11 +18,17 @@ Mozilla gecko.
 
 \section1 Using Sailfish WebView
 
-Currently you have to explicitly link against qtmozembed-qt5 at build time, to
-avoid a symbol issue which leads to a crash:
+Currently you have to explicitly link against qtmozembed-qt5 at build time.
+For this you need to modify both your spec file, and your pro file. In
+the spec file you should add line:
+
+\code
+BuildRequires:  pkgconfig(qt5embedwidget)
+\endcode
+
+And in your pro file you should add line:
 
 \code
-CONFIG += link_pkgconfig
```

Comment:
As you mentioned, this is needed if you do not sailfishapp. Good to promote usage of that as well.

Reply:
At the same having a CONFIG twice doesn't really harm.

Reply:
It does harm - it doesn't work. You can't have both sailfishapp and link_pkgconfig there.

Reply:
But I'll rewrite this section so that it mentions both

Reply:
Thank @vigejolla :+1: